d2l-scroll-wrapper.js

```diff
@@ -149,20 +137,32 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-scroll-wrapper">
 			:host([h-scrollbar]) .wrapper {
 				@apply --d2l-scroll-wrapper-h-scroll;
```

Comment:
Can the usages of this and the ones below not be removed yet because somewhere overriding the mixins needs to have the `show-actions` added?

Reply:
Or is this because of the margin stuff in rubric?

Reply:
Just the margin stuff in rubrics. :(

Reply:
Ok :( So hopefully not for long

Reply:
Yeah I plan to sort through them when I go in there to remove the border stuff and then follow up here to remove them entirely.